.ci/build_nccl_rdma_sharp_plugins.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -leE
 
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+cd ${SCRIPT_DIR}
```

Comment:
why needed

Reply:
This is needed because of `TOP_DIR="$(git rev-parse --show-toplevel)"` in `settings.sh`. The scripts may be called within another git repo so TOP_DIR may refer to incorrect place.